build/go_check.sh

```diff
@@ -26,6 +26,15 @@ if [[ "${f}" ]]; then
   exit 1
 fi
 
+# Check dependencies up-to-date
+go mod download
```

Comment:
Is this necessary?

Reply:
Yea, because in the container, the dependencies are not installed, so without running this first. go get -u all will always print stuff.